src/utils/reviewAlert.ts

```diff
@@ -1,50 +1,95 @@
-import AsyncStorage from '@react-native-async-storage/async-storage';
 import lang from 'i18n-js';
+import * as ls from '@/storage';
 import { Linking, NativeModules } from 'react-native';
 import { WrappedAlert as Alert } from '@/helpers/alert';
-const { RainbowRequestReview } = NativeModules;
+import { ReviewPromptAction } from '@/storage/schema';
+import { IS_IOS } from '@/env';
+import { logger } from '@/logger';
+
+const { RainbowRequestReview, RNReview } = NativeModules;
 
 export const AppleReviewAddress =
   'itms-apps://itunes.apple.com/us/app/appName/id1457119021?mt=8&action=write-review';
+export const PlayStoreAddress =
+  'market://details?id=me.rainbow&showAllReviews=true';
+
+const TWO_MONTHS = 1000 * 60 * 60 * 24 * 60;
+
+export const numberOfTimesBeforePrompt: {
+  [key in ReviewPromptAction]: number;
+} = {
+  UserPrompt: 0, // this should never increment
+  TimesLaunchedSinceInstall: 5,
+  SuccessfulFiatToCryptoPurchase: 3,
+  DappConnections: 4,
```

Comment:
After how many dApp connections should we prompt?

Reply:
q for seyi but id say lower unless this also tracks txs, if its just connections id do 2